android/java/org/chromium/chrome/browser/brave_leo/BraveLeoPlansActivity.java

```diff
@@ -48,6 +49,13 @@ protected void triggerLayoutInflation() {
         mMonthlySubscriptionAmountText = findViewById(R.id.monthly_subscription_amount_text);
         mUpgradeButton = findViewById(R.id.tv_upgrade_now);
 
+        TextView refreshCredentialsButton = findViewById(R.id.refresh_credentials_button);
+        refreshCredentialsButton.setOnClickListener(
+                v -> {
+                    BraveVpnUtils.startActivityBraveAccount(
+                            BraveLeoPlansActivity.this, BraveVpnUtils.getBraveAccountRecoverUrl());
```

Comment:
does it use `product=vpn` in the URL for Leo as well? It doesn't seem right to me.